pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "mm_image_processing"
+version = "0.1.0"
+packages = [] # add required pacakges later?
+description = "IP service for MM team in E4E"
+authors = ["E4E - Mangrove Monitoring SWE Team"]
+
+# poetry entrypoint for running our Flask server
+[tool.poetry.scripts]
+image-processing-server = "mm_image_processing.server:main"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+Flask = "2.3.1"
+requests = "2.28.2"
+boto3 = "1.26.120"
+numpy = "1.24.3"
+rasterio = "1.3.6"
+rioxarray = "0.14.1"
+xarray = "2023.4.2"
+distancerasters = "0.3.2"
+opencv-contrib-python = "4.7.0.72"
+dask = "2023.4.0"
+simplejson = "3.19.1"
+Flask-Cors = "3.0.10"
+flask-sock = "0.6.0"
+h11 = "0.14.0"
+websockets = "11.0.2"
+geopandas = "0.12.2"
+pandas = "1.5.3"
```

Comment:
`pandas` is a dependency of `geopandas` - are you explicitly using `pandas` in this project?

`1.5.3` is very limiting - did you mean `^1.5`?

Reply:
I don't believe so, i'll remove it for now and if we do end up requiring pandas in the future we can re-add it to the toml file.